cli.py

```diff
@@ -75,14 +75,14 @@ def display_station(station, geocode=False, use_colors=False):
     padding = ' ' * 1
     total_slots = station.get('extra', {}).get('slots')
     if not total_slots:
-        if station['empty_slots']:
+        if station.get('empty_slots') is not None:
             total_slots = station['free_bikes'] + station['empty_slots']
         else:
             total_slots = 0
 
     total_slots = float(total_slots)
```

Comment:
Do you think this would work out the same?

```
bikes = station['free_bikes']
slots = station['empty_slots']

if 'slots' in station.get('extra', {}):
    total_slots = float(station['extra']['slots'])
elif bikes is not None and slots is not None:
    total_slots = float(bikes + free)
else:
    total_slots = 0.0
```

I think it clears it up a bit